frontend/README.md

```diff
@@ -3,12 +3,11 @@
 > For architecture, see `architecture.md`
 
 ## Install
-- `$ apt install nodejs npm`
-- `$ npm i`
+ `$ apt install nodejs npm`
+ `$ npm i`
```

Comment:
I normally do this:

````suggestion
```console
apt install nodejs npm
npm install
```
````

Also, I make a list of prerequisite software and refer to the installation instructions because they change all the time and every major one like Node.js and npm have their preferred way to install. I also try to prevent using abbreviations.